src/wp-includes/class-wp-script-modules.php

```diff
@@ -949,4 +949,12 @@ public function print_a11y_script_module_html() {
 			. '<div id="a11y-speak-polite" class="a11y-speak-region" aria-live="polite" aria-relevant="additions text" aria-atomic="true"></div>'
 			. '</div>';
 	}
+
+	public function clone_without_enqueued_modules(): WP_Script_Modules {
```

Comment:
Relatedly, I just learned that PHP 8.5 has `clone` with support for setting properties on a cloned object: https://www.php.net/releases/8.5/en.php#clone-with